sf-docker/.env.sfac

```diff
@@ -21,7 +21,7 @@ FilterConfiguration__BlacklistsConfiguration__Blacklists__2=third_black_list_id
 FilterConfiguration__FaceMaskConfiguration__Enabled=false
 FilterConfiguration__FaceMaskConfiguration__DenyingDebounceMs=4000
 
-FilterConfiguration__IntentionalAccessConfiguration__Enabled=true
+FilterConfiguration__IntentionalAccessConfiguration__Enabled=false
```

Comment:
discused with MVI, this shoudl definitely by turn off by default
PR to SF AC will be next